src/state/assets/userAssets.ts

```diff
@@ -329,10 +345,29 @@ export const createUserAssetsStore = (address: Address | string) =>
               userAssets: userAssetsMap,
             };
         }),
+
+      hiddenAssets: new Set<UniqueId>(),
+
+      getHiddenAssetsIds: () => Array.from(get().hiddenAssets),
+
+      setHiddenAssets: (uniqueIds: UniqueId[]) => {
+        set(prev => {
+          const hiddenAssets = new Set(prev.hiddenAssets);
+          uniqueIds.forEach(uniqueId => {
+            if (hiddenAssets.has(uniqueId)) {
+              hiddenAssets.delete(uniqueId);
+            } else {
+              hiddenAssets.add(uniqueId);
+            }
```

Comment:
should this function be named `toggleHiddenAsset`?

Reply:
Eh, I like keeping the same nomenclature as the rest of the file personally. Pass it a list of uniqueIds and have it set it to either hidden or unhide it.